### PR TITLE
single values in operation nodes

### PR DESCRIPTION
**Pull request by jeaunrg**
Add parameters in division, multiplication, addition and substraction to be able to do operation between the reference image and a single value

### CODE REVIEW

File: src/controller/modules_fn.py [hunk 1]
```diff
@@ -106,11 +106,19 @@ def addImages(widget):
...
-        if parent_name not in _IMAGES_STACK.keys():
-            return print("'{}' not in image stack".format(parent_name))
-    im = core.addImages([_IMAGES_STACK[parent_name] for parent_name in parent_names])
+    if widget.singleValue.isChecked():
+        try:
+            value = float(widget.value.text())
+        except ValueError:
+            return print("addition value ERROR")
+        ref_parent_name = widget.reference.currentText()
+        im = core.substractImages(_IMAGES_STACK[ref_parent_name], value=value)
```
Comment (Hboni) [inline src/controller/modules_fn.py:115]: I thinks that's a mistake, we are in addition no?

File: src/controller/modules_fn.py [hunk 2]
```diff
@@ -135,11 +150,19 @@ def multiplyImages(widget):
...
-        if parent_name not in _IMAGES_STACK.keys():
-            return print("'{}' not in image stack".format(parent_name))
-    im = core.multiplyImages([_IMAGES_STACK[parent_name] for parent_name in parent_names])
+    if widget.singleValue.isChecked():
+        try:
+            value = float(widget.value.text())
+        except ValueError:
+            return print("multiply value ERROR")
+        ref_parent_name = widget.reference.currentText()
+        im = core.divideImages(_IMAGES_STACK[ref_parent_name], value=value)
```
Comment (Hboni) [inline src/controller/modules_fn.py:159]: Same error, we are in multiplication no?

File: src/controller/modules_fn.py (hunk 1 above)
Comment (Hboni) [inline src/controller/modules_fn.py:115]: I am not sure about that, but is `_IMAGES_STACK[ref_parent_name]` really a list?
According to line 121 I would say no, but you must know better ;)

**Review by Hboni — CHANGES_REQUESTED** [review]
Some typo errors

File: src/controller/modules_fn.py (hunk 1 above)
Comment (jeaunrg) [reply src/controller/modules_fn.py:118]: My mistake

File: src/controller/modules_fn.py (hunk 2 above)
Comment (jeaunrg) [reply src/controller/modules_fn.py:162]: RE My mistake

File: src/controller/modules_fn.py (hunk 1 above)
Comment (jeaunrg) [reply src/controller/modules_fn.py:118]: No its an unique numpy array, as requested by the first argument of core.substractImages function

Comment (Hboni) [reply src/controller/modules_fn.py]: I was thinking about addition here, now it's corrected I put my comment on the real line

File: src/controller/modules_fn.py [hunk 3]
```diff
@@ -106,11 +109,19 @@ def addImages(widget):
...
-        if parent_name not in _IMAGES_STACK.keys():
-            return print("'{}' not in image stack".format(parent_name))
-    im = core.addImages([_IMAGES_STACK[parent_name] for parent_name in parent_names])
+    if widget.singleValue.isChecked():
+        try:
+            value = float(widget.value.text())
+        except ValueError as e:
+            return print(e)
+        ref_parent_name = widget.reference.currentText()
+        im = core.addImages(_IMAGES_STACK[ref_parent_name], value=value)
```
Comment (Hboni) [inline src/controller/modules_fn.py:118]: COPY: I am not sure about that, but is `_IMAGES_STACK[ref_parent_name]` really a list?
According to line 124 I would say no, but you must know better ;)

Comment (jeaunrg) [reply src/controller/modules_fn.py:118]: Oups youre right the first argument is always a list for multiplication and addition..

**Review by Hboni — APPROVED** [review]

**Pull request merged**